pyproject.toml

```diff
@@ -59,7 +59,12 @@ repository = "https://github.com/pymc-labs/pymc-marketing"
 #changelog = ""
 
 [tool.ruff.lint]
-select = ["E", "F", "I", "W", "UP"]
+select = ["E", "F", "I", "W", "UP", "RUF"]
+ignore = [
+    "RUF001", # String contains ambiguous character (such as Greek letters)
+    "RUF002", # Docstring contains ambiguous character (such as Greek letters)
+    "RUF012", # Mutable class attributes should be annotated with `typing.ClassVar`
+]
```

Comment:
For the test suite, you might want you use something file specific ruff rules. For instance, the assert statements

```toml
[tool.ruff.lint.per-file-ignores]
"__init__.py" = ["F401"]
"**/tests/*" = ["S101"]

Reply:
Ok! Actually these ignores I took from the PyMC config as they appear on the code base. I'll create a tickett to improve the rules in the test suite!